system/init.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+spl_autoload_register(function($className) {
+  require_once 'classes/' . $className . '.php';
+});
+
+// Initialized the route object
+$route_obj = new route;
```

Comment:
It's better to use camel case for `$rout_obj` as we follow the OOPS approach for the whole project.